packages/date-time/src/components/Calendar/Calendar.styles.ts

```diff
@@ -25,12 +27,22 @@ export const styles = (props: ICalendarStyleProps): ICalendarStyles => {
     divider: {
       top: 0,
       borderRight: '1px solid',
-      borderColor: palette.neutralLight
+      borderColor: palette.neutralLight,
+      selectors: {
+        [MinimumScreenSelector]: {
+          display: 'none'
+        }
+      }
     },
     monthPickerWrapper: [
```

Comment:
This is scary to me. It looks like your fix is, if the user is a certain screen size, just hide the month picker part entirely.

On your example page, go down to "Inline Calendar with month picker and no day picker".
![image](https://user-images.githubusercontent.com/8549582/60224278-99f2e880-9836-11e9-9ed0-ca007bc11930.png)
It's gone, because you're hiding it.

On the example "Inline calendar with overlayed month picker", click on the header button "June 2019", and the same thing happens, the whole control disappears.

Reply:
It looks like the behavior you want is to have the month picker be overlayed all the time, rather than trying to display next to the day picker. Have you considered just passing in props to the control to achieve this? the prop 'showMonthPickerAsOverlay' achieves exactly this, without the accompanying issues. Or, if you really don't want the month picker, you can use 'isMonthPickerVisible: false' to remove it entirely, which will also get rid of the divider.

Reply:
If this needs to be dynamic, I wonder if there's a way to adjust the state of the actual rendered content based on screen size. That way, if the screen just isn't big enough, we can pretend the user passed in these props and just do the overlay solution by default. @dzearing, are you familiar with any way to do this in Fabric? Or do we have to have a pure CSS solution here?

Reply:
@lorejoh12 thanks for the comment, and would like to understand a better way to fix this. I was going off of this implementation in the `office-ui-fabric-react` package, where we hide the month picker on smaller screens.
https://github.com/OfficeDev/office-ui-fabric-react/blob/6ff7873e60436513ce7c3e4f39982a461fa5c86f/packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss#L721-L728

Reply:
There's a reason I ended up stripping that part out when I created the new date-time Calendar. It's fragile and hard to get it right, since it causes problems like the ones you're seeing when we aren't super-careful with it, and you end up having the component's rendered result not match the props that got passed in.
 
If this is only going to be used for mobile devices and you always want to have the overlay, I'd definitely suggest just passing in the appropriate props.
 
If you really need this to be dynamic, a better solution might be to add a new getter function to Calendar.base.tsx (and Calendar.tsx for the office-ui-fabric-react one), "showingMonthPickerAsOverlay", where we can do something like:
private showingMonthPickerAsOverlay = () => {
   return window.screen.size < MINIMUM_SCREEN_SIZE || this.props.showMonthPickerAsOverlay;
},
 
and then replace all references to showMonthPickerAsOverlay with it. It will effectively override the prop when the screen is too small. If we want it to work on screen resizes also we'd have to add window.onResize handlers

Reply:
onresize = () => {
this.setState({screenWidth: getWindow().screen.width })
}

and you can read the state in the "showingMonthPickerAsOverlay" function, return this.state.screenWidth < MINIMUM_SCREEN_SIZE and that way it recomputes when the component rerenders due to a state change